gateway-service/src/main/resources/configs/common/application.conf

```diff
@@ -87,3 +87,8 @@ scopeFiltersConfig = [
     ]
   }
 ]
+
+log.config = {
```

Comment:
Config is specific to EntitySevice, lets move it under the entity service level.
`entity.service.log.config = { query.threshold.millis = 1500}`

And, do we need hierarchy here - query.threashold.millis?
or we can simply have `slowQueryThresholdMs = 1500` ? 
``entity.service.log.config = { slowQueryThresholdMs = 1500}`

Reply:
will move to entity.service.log.config. 

The hierarchical naming is more flexible for future expansion, say there's another query logging configuration then, it can follow the same pattern. Also, it's still matches with HOCON convention. Do you see a downside in keeping hierarchical?

Reply:
Actually, should it be only for entity.service? Should the other services have access to this configuration as well?